doc/FAQ.md

```diff
@@ -119,3 +119,46 @@ we should provide it like this:
 ```java
 NO_PROXY: localhost,127.0.0.1,.google.com,.github.com
 ```
+
+### Optimistic Locking Behavior
+
+Unfortunately it's a little complicated as it depends on what operation you are doing - we'll work towards ensuring the Javadocs are as informative as possible.  Here is quick overview:
+
+- Basic mutative operations such as update and all variations of patch (patch, edit, accept, serverSideApply) that operate on a given item - are all locked to the resourceVersion on that item.  If you don't want this behavior then set the resourceVersion to null:
```

Comment:
Maybe a sentence or a link to a reference on the locking behavior in Kubernetes would help?

Reply:
Sure, there will be another pr to more throughly cover server side apply, that can be part of it.